robottelo/cli/subscription.py

```diff
@@ -69,7 +69,7 @@ def refresh_manifest(cls, options=None):
         return result
 
     @classmethod
-    def manifest_history(cls, options=None, expect_csv=True):
+    def manifest_history(cls, options=None):
```

Comment:
Should `output_format='csv'` somehow be specified here? I ask because I see that you're dropping the `expect_csv` argument.